src/js/view/view.js

```diff
@@ -558,6 +558,7 @@ function View(_api, _model) {
             const provider = model.getVideo() || instreamProvider;
             if (provider && provider.setFullscreen) {
                 provider.setFullscreen(state);
+                toggleClass(_playerElement, 'jw-flag-ios-fullscreen', state);
```

Comment:
This class should be added in response to the native fullscreen event, not following the request to go fullscreen.